docker-release.sh

```diff
@@ -31,15 +42,16 @@ docker login -u $user -p $pass
 # Remove earlier builds
 for dbtag in "${dbtags[@]}"
 do
+  name="$(containerName $dbtag)"
```

Comment:
I think this is going to fail: 
1. Space in the name
2. `tinode alldbs` won't exists because you names it just `tinode`

Reply:
No, `containerName` is a function.
`$(containerName $dbtag)` will evaluate to `tinode`.